<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C02VD330HTD6" ignored="4" name="Command line test" tests="27" failures="9" timestamp="24 Aug 2018 21:52:23 GMT" time="0.252" errors="0">
  <testcase name="@BeforeClass beforeClass" time="0.002" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.ApiClient.createExperiment(ApiClient.java:42)
at com.databricks.mlflow.client.MultiThreadedTest.beforeClass(MultiThreadedTest.java:20)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 50 more
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeClass -->
  <testcase name="addGetRun" time="0.195" classname="com.databricks.mlflow.client.ApiClientTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.ApiClient.createExperiment(ApiClient.java:42)
at com.databricks.mlflow.client.ApiClientTest.addGetRun(ApiClientTest.java:54)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- addGetRun -->
  <testcase name="createExistingExperiment" time="0.001" classname="com.databricks.mlflow.client.ApiClientTest">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.ApiClient.createExperiment(ApiClient.java:42)
at com.databricks.mlflow.client.ApiClientTest.createExistingExperiment(ApiClientTest.java:27)
... 29 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- createExistingExperiment -->
  <testcase name="getCreateExperimentTest" time="0.001" classname="com.databricks.mlflow.client.ApiClientTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.ApiClient.createExperiment(ApiClient.java:42)
at com.databricks.mlflow.client.ApiClientTest.getCreateExperimentTest(ApiClientTest.java:19)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getCreateExperimentTest -->
  <testcase name="listExperimentsTest" time="0.001" classname="com.databricks.mlflow.client.ApiClientTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller._get(HttpCaller.java:60)
at com.databricks.mlflow.client.HttpCaller.get(HttpCaller.java:50)
at com.databricks.mlflow.client.ApiClient.listExperiments(ApiClient.java:37)
at com.databricks.mlflow.client.ApiClientTest.listExperimentsTest(ApiClientTest.java:33)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- listExperimentsTest -->
  <testcase name="checkParamsAndMetrics" time="0.0" classname="com.databricks.mlflow.client.ApiClientTest">
    <skipped/>
  </testcase> <!-- checkParamsAndMetrics -->
  <testcase name="checkSearchMetrics" time="0.0" classname="com.databricks.mlflow.client.ApiClientTest">
    <skipped/>
  </testcase> <!-- checkSearchMetrics -->
  <testcase name="checkSearchMixed" time="0.0" classname="com.databricks.mlflow.client.ApiClientTest">
    <skipped/>
  </testcase> <!-- checkSearchMixed -->
  <testcase name="testSearchParameters" time="0.0" classname="com.databricks.mlflow.client.ApiClientTest">
    <skipped/>
  </testcase> <!-- testSearchParameters -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="testRunsInParallel" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- testRunsInParallel -->
  <testcase name="checkNumberOfRunIds" time="0.0" classname="com.databricks.mlflow.client.MultiThreadedTest">
    <skipped/>
  </testcase> <!-- checkNumberOfRunIds -->
  <testcase name="createExperiment_IllegalJsonSyntax" time="0.002" classname="com.databricks.mlflow.client.HttpTest">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.HttpTest.createExperiment_IllegalJsonSyntax(HttpTest.java:18)
... 29 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 48 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- createExperiment_IllegalJsonSyntax -->
  <testcase name="createExperiment_MissingJsonField" time="0.002" classname="com.databricks.mlflow.client.HttpTest">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller.post(HttpCaller.java:87)
at com.databricks.mlflow.client.ApiClient.post(ApiClient.java:117)
at com.databricks.mlflow.client.HttpTest.createExperiment_MissingJsonField(HttpTest.java:24)
... 29 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 48 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- createExperiment_MissingJsonField -->
  <testcase name="getExperiment_NonExistentId" time="0.003" classname="com.databricks.mlflow.client.HttpTest">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.databricks.mlflow.client.HttpServerException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller._get(HttpCaller.java:60)
at com.databricks.mlflow.client.HttpCaller.get(HttpCaller.java:50)
at com.databricks.mlflow.client.ApiClient.get(ApiClient.java:113)
at com.databricks.mlflow.client.HttpTest.getExperiment_NonExistentId(HttpTest.java:13)
... 29 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- getExperiment_NonExistentId -->
  <testcase name="nonExistentPath" time="0.001" classname="com.databricks.mlflow.client.HttpTest">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.databricks.mlflow.client.HttpClientException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.databricks.mlflow.client.HttpClientException but got org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller._get(HttpCaller.java:60)
at com.databricks.mlflow.client.HttpCaller.get(HttpCaller.java:50)
at com.databricks.mlflow.client.ApiClient.get(ApiClient.java:113)
at com.databricks.mlflow.client.HttpTest.nonExistentPath(HttpTest.java:8)
... 29 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- nonExistentPath -->
  <testcase name="checkExperiment" time="0.001" classname="com.databricks.mlflow.client.PythonScikitLearnTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connect to localhost:5001 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:394)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at com.databricks.mlflow.client.HttpCaller._get(HttpCaller.java:60)
at com.databricks.mlflow.client.HttpCaller._get(HttpCaller.java:54)
at com.databricks.mlflow.client.ApiClient.getExperiment(ApiClient.java:33)
at com.databricks.mlflow.client.PythonScikitLearnTest.checkExperiment(PythonScikitLearnTest.java:21)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
at java.net.PlainSocketImpl.socketConnect(Native Method)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:589)
at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
... 49 more
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- checkExperiment -->
  <testcase name="checkGetArtifact" time="0.0" classname="com.databricks.mlflow.client.PythonScikitLearnTest">
    <skipped/>
  </testcase> <!-- checkGetArtifact -->
  <testcase name="checkListArtifacts" time="0.0" classname="com.databricks.mlflow.client.PythonScikitLearnTest">
    <skipped/>
  </testcase> <!-- checkListArtifacts -->
  <testcase name="checkParamsAndMetrics" time="0.0" classname="com.databricks.mlflow.client.PythonScikitLearnTest">
    <skipped/>
  </testcase> <!-- checkParamsAndMetrics -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="com.databricks.mlflow.client.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="com.databricks.mlflow.client.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="com.databricks.mlflow.client.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="com.databricks.mlflow.client.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
</testsuite> <!-- Command line test -->
